staging/src/k8s.io/apiserver/pkg/features/kube_features.go

```diff
@@ -158,7 +158,7 @@ var defaultKubernetesFeatureGates = map[featuregate.Feature]featuregate.FeatureS
 	ServerSideApply:         {Default: true, PreRelease: featuregate.Beta},
 	StorageVersionHash:      {Default: true, PreRelease: featuregate.Beta},
 	WatchBookmark:           {Default: true, PreRelease: featuregate.GA, LockToDefault: true},
-	APIPriorityAndFairness:  {Default: false, PreRelease: featuregate.Alpha},
+	APIPriorityAndFairness:  {Default: true, PreRelease: featuregate.Beta},
```

Comment:
we can hold off from making this beta right now -- https://github.com/kubernetes/test-infra/pull/19757 should allow us to test this feature as it is in alpha

Reply:
reverted